doc/cabal-project.rst

```diff
@@ -194,8 +194,8 @@ Specifying Packages from Remote Version Control Locations
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Starting with Cabal 2.4, there is now a stanza
-``source-repository-package`` for specifying packages from an external
-version control.
+``source-repository-package`` for specifying remote packages from a version control system that cabal should consider during package retrieval.
+This allows use of a remote package in version control, rather than looking for a package in Hackage.
```

Comment:
Again "remote package" is a bit problematic. "package in a remote versioning system" sounds less problematic.

Reply:
Indeed, I have some local packages stored in local version control. They're not _currently_ slated for use by cabal, but I could see it happening.